common/src/main/java/tachyon/Constants.java

```diff
@@ -319,6 +319,24 @@
   public static final String USER_FILE_WAITCOMPLETED_POLL_MS =
       "tachyon.user.file.waitcompleted.poll.ms";
 
+  //tachyo-fuse related conf keys
+
+  /**
+   * Passed to fuse-mount, maximum granularity of wirte operations:
+   * Capped by the kernel to 128KB max (as of Linux 3.16.0).
+   */
+  public static final String FUSE_MAXWRITE_BYTES = "tachyon.fuse.maxwrite.bytes";
+  /**
+   * Have the fuse process log every FS request
+   */
+  public static final String FUSE_DEBUG_ENABLE = "tachyon.fuse.debug.enable";
```

Comment:
better to be 
`FUSE_DEBUG_ENABLED = "tachyon.fuse.debug.enabled"` to be consistent with other flags.